src/main/scala/geotrellis/sdg/Grump.scala

```diff
@@ -97,7 +97,7 @@ object Grump {
   def masksForBoundary(
     rdd: RDD[Geometry],
     layout: LayoutDefinition,
-    boundary: MultiPolygon,
+    boundary: Geometry,
```

Comment:
Huh. This is in Grump. Was this a change that snuck in from some unrelated work?

Reply:
Buffering `MultiPolygon` can potentially join two polygons together and result in `Polygon`. In JTS API that is expressed in usual up-cast to `Geometry` as result of `buffer` method. We used to need to match on the result to figure out if we got a `Polygon` or `MultiPolygon` back. But now its just less hassle to propagate the `Geometry` throughout the code, which is a thing that happens.